examples/spatial.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>videojs-vr Demo</title>
+   <link href="../node_modules/video.js/dist/video-js.css" rel="stylesheet">
+   <link href="../dist/videojs-vr.css" rel="stylesheet">
+</head>
+<body>
+  <video width="640" height="300" id="videojs-vr-player" class="video-js vjs-default-skin" controls playsinline>
+    <source src="your-spatial-video.mp4" type="video/mp4">
```

Comment:
Do you have any examples that we could use in the repo? it makes it much easier to test.

Reply:
I guess we can use example from Omnitone library. For example this one: https://storage.googleapis.com/omnitone-demo.google.com.a.appspot.com/fuerza-imprevista-1080p-h264-aac.mp4 any objections?

Reply:
Linking to it should be no problem. If we wanted to include it in the repo, we'd need to consider the license.

Reply:
It looks like we can't link it because of CORS error. I can test locally only after downloading the file but to add it to the repo sources we will need to have `Ok` from the artist.

Reply:
using `crossorigin` on the video element appears to work for that video.